.github/workflows/release.yml

```diff
@@ -0,0 +1,77 @@
+name: release
+on:
+  push:
+    tags:
```

Comment:
Hi @jagedn , 

So basically we'll just tag the plugin locally and then push the plugin to trigger the build process of artifacts right?

Reply:
exactly , this is the idea

and not only locally, also you can use the web interface

Reply:
Ah, gotcha - then let's roll! 🛼